src/mobile/src/main/java/tupperdate/android/profile/Profile.kt

```diff
@@ -0,0 +1,129 @@
+package tupperdate.android.profile
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.ScrollableColumn
+import androidx.compose.foundation.Text
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.ui.tooling.preview.Preview
+import dev.chrisbanes.accompanist.coil.CoilImage
+import tupperdate.android.R
+import tupperdate.android.ui.TupperdateTheme
+import tupperdate.android.ui.material.BrandedButton
+
+@Composable
+fun Profile(
+    onCloseClick: () -> Unit,
+    onEditClick: () -> Unit,
+    onSaveClick: () -> Unit,
+    onSignOutClick: () -> Unit
+) {
+    ScrollableColumn(Modifier.fillMaxSize().padding(16.dp)) {
+        Row(
+            Modifier.fillMaxWidth(),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            Text(
+                text = stringResource(id = R.string.profile_title),
+                style = MaterialTheme.typography.subtitle1
+            )
+            IconButton(onClick = { onCloseClick() }) {
+                Icon(
+                    asset = vectorResource(id = R.drawable.ic_home_dislike_recipe),
+                    tint = Color.Black
+                )
+            }
+        }
+
+        Row(
+            Modifier.padding(top = 40.dp, bottom = 37.dp)
+                .align(Alignment.CenterHorizontally)
+        ) {
+            Column(horizontalAlignment = (Alignment.CenterHorizontally)) {
+                CoilImage(
+                    data = "https://images.firstpost.com/wp-content/uploads/2019/04/thor380.jpg",
+                    modifier = Modifier.size(96.dp).clip(CircleShape)
+                        .border(2.dp, Color.Gray, CircleShape),
+                    contentScale = ContentScale.Crop
+                )
+                Button(
+                    onClick = { onEditClick() },
+                    colors = ButtonConstants.defaultButtonColors(
+                        contentColor = Color.Black,
+                        backgroundColor = Color.Transparent
+                    ),
+                    elevation = ButtonConstants.defaultElevation(0.dp)
+                ) {
+                    Text(
+                        text = stringResource(id = R.string.profile_editpic), fontSize = 7.sp
```

Comment:
Should apparently be 10dp, not 7. Instead of defining it here, we could refer to a text style from the theme